### ISSUE TITLE
import rmarkdown fails to create notebook

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud.rmd#16]
_From @gordonwoodhull on November 11, 2016 3:19_

Advanced/Import from Rmarkdown fails with the following error:

```
Error: importRmd: Error in fun(...) : failed to create new notebook
R trace:
stop("failed to create new notebook")
fun(...)
at success (http://127.0.0.1:8080/js/rcloud_bundle.js:142:23)
From previous event:
at Object.importRmd (http://127.0.0.1:8080/js/rcloud_bundle.js:148:54)
at HTMLSpanElement.do_import (rcloud.rmd.js:55:40)
at HTMLSpanElement.dispatch (http://127.0.0.1:8080/lib/js/jquery-2.1.1.js:4409:9)
at HTMLSpanElement.elemData.handle (http://127.0.0.1:8080/lib/js/jquery-2.1.1.js:4095:28)
```

_Copied from original issue: att/rcloud.flexdashboard#10_

**Comment by gordonwoodhull** [comment]
@gaborcsardi, you're off the hook - just logging this because this is the first time I tested this side of things, and it's been a long day.

**Comment by gordonwoodhull** [comment]
_From @gaborcsardi on November 11, 2016 8:51_

@gordonwoodhull do you have an example notebook?

**Comment by gordonwoodhull** [comment]
_From @gaborcsardi on November 11, 2016 8:51_

Btw. the import/export is in the `rcloud.rmd` package I think.

**Comment by gordonwoodhull** [comment]
Okay, I'm not able to repro on my work machine. Maybe there is something odd about my laptop configuration, or something weird about the notebook I was working with.

**Comment by gaborcsardi** [comment]
Maybe it is the package versions? Ideally we would need to check the version of `knitr` I think.

**Comment by gordonwoodhull** [comment]
I think this is the same as #18.

**Comment by gordonwoodhull** [comment]
fixed by #22
